cpp/daal/src/algorithms/svm/svm_train.h

```diff
@@ -0,0 +1,129 @@
+/* file: svm_train.h */
```

Comment:
bad name file, confused with public. maybe?
```suggestion
/* file: svm_inner_batch.h */
```